README.md

```diff
@@ -14,6 +14,10 @@ A collection may contain metadata that identifies these versions.
 PEP440 is the schema used to describe the versions of Ansible.
 <!--end requires_ansible-->
 
+## Python version compatibility
+
+As the `aiohttp` for Python has [requirement](https://github.com/aio-libs/aiohttp#requirements) of Python 3.7 or greater, this collection requires Python 3.7 or greater.
```

Comment:
Actually, it's py3.6, not 3.7.

Reply:
aiohttp requires 3.7 and onwards.

Reply:
The stable version requires Python 3.6, development 3.7. But none of this answers the question from my review: why was Python 3 selected as the minimum version while most other community collections still need to maintain compatibility with 2.7 if at all possible.

Reply:
@tadeboro Due to the requirements of the dependent library (aiohttp).  Most of the cloud-related collections are 3-only due to dependencies, my understanding is that documenting this should be adequate for inclusion.

Reply:
@Akasurde in the community meeting we discussed this and the confusion is why we are requiring aiohttp over other http connection methods available in core anisble.  Can you please update this PR to include a note to the effect that we require python3, because we require aiohttp, because it adds major performance improvements?

Reply:
Were collection requirements known at the time when the collection was initially designed? If yes, then aiohttp library could possibly be avoided and the collection made Python 2.7 compatible. Otherwise, the dependency argument holds and things are all good.

Reply:
The 1.0.0 release of the collection was Oct 2020 and development started no later than Feb 2020 (I'm pretty sure it was even before then though).  We started working on inclusion requirements around Aug 2020 I think?   But didn't really get the Python requirements until March 2021.

Reply:
Thanks @jillr! I will ad this to info to my review and stop nagging you all about Python versions ;)